### ISSUE TITLE
Add flag for accepting single or comma separated input values

### ISSUE THREAD

**Issue by forgedhallpass** [projectdiscovery/httpx#737]
Currently `httpx` only accepts input through piping or from a file. Adding a new option to accept a single input would be convenient in certain cases (e.g. debugging).

```yaml
INPUT:
   -u, -target string      input target host to probe
```

**Comment by ariasmn** [comment]
Hi! 

Wanted to contribute to this amazing project (that a friend of mine heavily uses :+1: ). Just opened a [PR for this issue](https://github.com/projectdiscovery/httpx/pull/759)
As I commented, there might be a couple of things that need some work. Please let me know in that case!

**Comment by ariasmn** [comment]
Closed with #759
